### DISCUSSION TITLE
Add statistics graph side by side with timeseries graph / or enable dashboard-row-panel selection of datasource for in row statistics when using datasource in repeated row panel.

### DISCUSSION THREAD

**Discussion by jpedrick** [grafana/grafana#38190] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Within the same graph panel, add the ability to have a histogram with n-buckets autofitted at even intervals between series min/max and simple statistics as small table(min, max, mean, stddev, possibly by user formula).

**Why is this needed**:
Currently when using a 'dashboard' data source with row repetition, there's no way to select which row to use for the source data. If a particular query is expensive, it would be helpful to be able to reuse the data that's already fetched in the graph-row to show summary statistics, histogram, etc.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
